fares-data-build-tool/fdbt-netex-output
# Implementation for odh uploader to use email from the generated netex

# Description

Add the ability to email the generated netex

# Testing instructions

-  ODH Uploader has the ability to get the email, uuid and other keys to add into the email. There is a an email template.

In order to test if you checkout the https://github.com/fares-data-build-tool/fdbt-dev/pull/12 and this pr. Under the fdbt-dev branch if you run `make generate-email-odh-uploader`

This will generate the periodMultiService

You can change this to see by changing the filename for the make generator command for example for single you can use

```
generate-email-odh-uploader:
	./scripts/trigger_odh_uploader.sh single
```

This will allow you to test the N/A for products and services as the array does not exist.

No email can be sent locally so you can console.log(mailOptions) from odh-handler
# Type of change


# Checklist:
